backend/src/director.py

```diff
@@ -4,12 +4,16 @@
 from src.agents import get_intent_agent, get_answer_agent
 from src.prompts import PromptEngine
 from src.supervisors import solve_all
+from src.utils import Config
+from src.agents.web_agent import WebAgent
 
 logger = logging.getLogger(__name__)
-
+config = Config()
 engine = PromptEngine()
 director_prompt = engine.load_prompt("director")
 
+# Create an instance of the WebAgent
+web_agent = WebAgent(config.web_agent_llm, config.web_agent_model)
```

Comment:
Why do we create a web agent here?

Reply:
I have removed this variable. I guess earlier I was trying some other way of calling web agent so created this variable but later changed the flow and forgot to remove it.